includes/frontend.php

```diff
@@ -1558,4 +1571,40 @@ private function process_more_tag( $content ) {
 
 		return force_balance_tags( $parts['main'] ) . $more_link;
 	}
+
+	public function register_dynamic_imports( $dynamic_import_manager ): void {
+		$dynamic_import_manager->register_dynamic_import(
+			'SwiperBase',
```

Comment:
@rami-elementor These are existing keys. I don't think we can change this names, because 3rd parties might have implemented them.

Specifically this name is used here: `elementorModules.frontend.handlers.SwiperBase`